vmsync/vmsync.go

```diff
@@ -0,0 +1,163 @@
+package vmsync
+
+import (
+	"app/base/logging"
+	"app/base/models"
+	"app/base/utils"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+)
+
+var (
+	logger    *logrus.Logger
+	BatchSize = utils.GetEnv("BATCH_SIZE", 5000)
+)
+
+func init() {
+	logLevel := utils.GetEnv("LOGGING_LEVEL", "INFO")
+	var err error
+	logger, err = logging.CreateLogger(logLevel)
+	if err != nil {
+		fmt.Println("Error setting up logger.")
+		os.Exit(1)
+	}
+	logger.SetFormatter(&logrus.TextFormatter{
+		FullTimestamp: true,
+	})
+}
+
+func syncCveMetadata() {
+	apiCveMap, err := getAPICves()
+
+	if err != nil {
+		logger.Fatalf("Unable to get CVEs from VMaaS: %s", err)
+	}
+	logger.Infof("CVEs in VMaaS: %d", len(apiCveMap))
```

Comment:
It basically says the same as the log[ just before](https://github.com/RedHatInsights/vuln4shift-backend/pull/5/files#diff-f03f754ac47dfa235847556581a3690e853ae0fc4e8659ffea4a01645cd45df7R87), in `getAPICves`

Reply:
deleted